05.User-resource.md

```diff
@@ -124,6 +154,28 @@ Is worth mentioning there's an additional endpoint nested under rooms that you c
 POST /v1/user/:userId/rooms/:roomId/unreadItems
 ```
 
+```
+{
+	"chat": [
+		"560ae5f495756f1402bc841d",
+		"560aeef6f4b61c106fb2a9f3",
+		"560aef2b552ed7913279df52",
+		"560aef4f081f3a9c044d7d61",
+		"560aef66081f3a9c044d7d67",
+		"560b167a967c1bad7852bc57"
+	]
+}
+```
+
+Try it from the CLI:
+```
+'demo api-access';
+{
+    "verb": "post",
+    "resource": "{{api_url}}/v1/user/:userId/rooms/:roomId/unreadItems"
+}
```

Comment:
Add in corresponding `data` property. See as example: https://github.com/gitterHQ/docs/blob/master/04.Messages-resource.md#send-a-message